# Enable Mediapipe Solutions
enable_right_hand: true
enable_left_hand: true
enable_pose: true
enable_face: false

# Training Parameters
min_epochs: 100
max_epochs: 2000
min_delta: 0.01
patience: 150
batch_size: 64
optimizer: Adam
learning_rate: 5e-4
loss_function: cross_entropy
train_set_size: 0.8
validation_set_size: 0.1
test_set_size: 0.1

# Torch Compilation
compilation_mode: default
torch_compilation: false
profiler: false
fast_dev_run: false

hydra:
  run:
    dir: /home/davide/ROS/ICRA_2023/src/Mediapipe/mediapipe_gesture_recognition -
      ICRA/data/outputs/${now:%Y-%m-%d}/${now:%H-%M-%S}
